src/lib/components/List/List.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './List.css';
+
+class List extends React.Component {
+  render() {
+    const listItems = React.Children.map(this.props.children,
+      child => React.cloneElement(child, {
+        modifier: this.props.modifier,
+      }),
+    );
+
+    let listType;
+    switch (this.props.modifier) {
+      case 'divided':
+        listType = 'p-list--divided';
+        break;
+      case 'inline':
+        listType = 'p-inline-list';
+        break;
+      case 'middot':
+        listType = 'p-inline-list--middot';
+        break;
+      case 'split':
+        listType = 'p-list--divided is-split';
+        break;
+      default:
+        listType = 'p-list';
+    }
+
+    return (
+      <ul className={listType}>
+        { listItems }
+      </ul>
+    );
+  }
+}
+
+List.defaultProps = {
+  modifier: null,
+};
+
+List.propTypes = {
+  children: PropTypes.node.isRequired,
+  modifier: PropTypes.oneOf(['divided', 'inline', 'middot', 'split']),
```

Comment:
null or default for p-list?

Reply:
@anthonydillon Is it necessary if 'modifier' is not a required prop?

Reply:
Ah, I see it's not required. Thanks.